opta.spec

```diff
@@ -8,7 +8,13 @@ block_cipher = None
 a = Analysis(['opta/cli.py'],
              pathex=[],
              binaries=[],
-             datas=[('./config', 'config'), ('roots.pem', 'grpc/_cython/_credentials/')] + collect_data_files('hcl2'),
+             datas=[
+               ('./config', 'config'),
+               ('roots.pem', 'grpc/_cython/_credentials/'),
+               ('opta/commands/init_templates/environment/aws/opta.yml', 'opta/commands/init_templates/environment/aws'),
```

Comment:
I believe you can just specify ` ('opta/commands/init_templates', 'opta/commands/init_templates'),`